randovania/games/prime2/json_data/header.json

```diff
@@ -948,8 +948,8 @@
                 "extra": {}
             },
             "NoSuits": {
-                "long_name": "Suitless Ingclaw/Ingstorm",
-                "description": "If this trick is enabled, Samus may be expected to traverse through Ingclaw and Ingstorm without the Dark Suit or the Light Suit. Instead of the appropriate suits, Energy Tanks may be provided by the logic as a means to pass these obstacles.",
+                "long_name": "Suitless Dark Aether",
+                "description": "If this trick is enabled, Samus may be expected to traverse through the Dark Aether with a very low amount of Health, or Ingclaw and Ingstorm without the Dark Suit or the Light Suit. Instead of the appropriate suits, Energy Tanks may be provided by the logic as a means to pass these obstacles.",
```

Comment:
`... traverse through the Dark Aether without a suit or with low energy. Higher levels of this trick expects going through Ingclaw or Ingstorm without Dark Suit or Light Suit, respectively. ...`